### ISSUE TITLE
Crash when trying to update ffupdater from inapp

### ISSUE THREAD

**Issue by nick-77** [Tobi823/ffupdater#331]
Stacktrace:
```
java.lang.IllegalArgumentException: Unrecognized stream
	at android.content.pm.PackageInstaller$Session.fsync(PackageInstaller.java:1178)
	at de.marmaro.krt.ffupdater.installer.impl.SessionInstaller.copyApkToSession(Unknown Source:91)
	at de.marmaro.krt.ffupdater.installer.impl.SessionInstaller.install(Unknown Source:31)
	at de.marmaro.krt.ffupdater.installer.impl.SessionInstaller.access$install(Unknown Source:0)
	at de.marmaro.krt.ffupdater.installer.impl.SessionInstaller$executeInstallerSpecificLogic$3.invokeSuspend(Unknown Source:39)
	at w3.a.resumeWith(Unknown Source:8)
	at i4.l0.run(Unknown Source:102)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:240)
	at android.os.Looper.loop(Looper.java:351)
	at android.app.ActivityThread.main(ActivityThread.java:8364)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
	Suppressed: i4.j0: [p1{Cancelling}@ee5d50, Dispatchers.Main]
Timestamp: 2023-02-02T19:01:40.505

```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.7.1 (127) release |
| Device | RMX3301 (RMX3301, RED8ACL1, taro) |
| Manufacturer | realme (realme) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 13 (SDK: 33) |
| OS | dg02-pool03-kvm45, root, release-keys, 1673257942000 |

**Comment by mshrey** [comment]
Stacktrace:
```
java.lang.IllegalArgumentException: Unrecognized stream
	at android.content.pm.PackageInstaller$Session.fsync(PackageInstaller.java:1180)
	at de.marmaro.krt.ffupdater.installer.impl.SessionInstaller.copyApkToSession(Unknown Source:91)
	at de.marmaro.krt.ffupdater.installer.impl.SessionInstaller.install(Unknown Source:31)
	at de.marmaro.krt.ffupdater.installer.impl.SessionInstaller.access$install(Unknown Source:0)
	at de.marmaro.krt.ffupdater.installer.impl.SessionInstaller$executeInstallerSpecificLogic$3.invokeSuspend(Unknown Source:39)
	at w3.a.resumeWith(Unknown Source:8)
	at i4.l0.run(Unknown Source:102)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8741)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
	Suppressed: i4.j0: [p1{Cancelling}@746f53c, Dispatchers.Main]
Timestamp: 2023-02-02T20:14:15.630

```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.7.1 (127) release |
| Device | SM-M536B (m53xnnxx, m53x, m53x) |
| Manufacturer | samsung (samsung) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 13 (SDK: 33) |
| OS | SWDK6923, dpi, release-keys, 1673856816000 | 

        
I had the same problem. And even after reinstalling the app, I have the same problem. And cannot install/update  any apps.

**Comment by Tobi823** [comment]
[ffupdater-release.zip](https://github.com/Tobi823/ffupdater/files/10570687/ffupdater-release.zip)
 
Maybe I fixed the problem. Can you @mshrey @nick-77 try this version?

**Comment by mshrey** [comment]
That was quick, and it seems to have fixed the problem! Cheers!

**Comment by Tobi823** [comment]
Thanks :) I will release the fix as version 77.7.2

**Comment by nick-77** [comment]
Thanks sir, it's fixed :)
